scripts/build-initrd.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+source scripts/common.sh
+
+readonly BUSYBOX_VER=1.34.0
+readonly BUSYBOX_DL="https://busybox.net/downloads/busybox-${BUSYBOX_VER}.tar.bz2"
+readonly BUSYBOX_ARCHIVE=busybox.tar.bz2
+
+function download_busybox()
+{
+    echo "Downloading Busybox version ${BUSYBOX_VER}..."
+    curl -L -o ${OUT}/${BUSYBOX_ARCHIVE} ${BUSYBOX_DL} || exit 1
+    echo "Extracting..."
+    tar -xjf ${OUT}/${BUSYBOX_ARCHIVE} -C ${OUT} || exit 1
+}
+
+function build_busybox()
+{
+   echo "Building Busybox..."
+   cp -f scripts/busybox.config ${OUT}/busybox-${BUSYBOX_VER}/.config
+   pushd ${OUT}/busybox-${BUSYBOX_VER}
+   make -j `nproc` && make install
+   cd _install
```

Comment:
Split the following commands into new function. Let's say, `generate_rootfs`.

Reply:
?

Reply:
Do I misunderstood something? Am I put the command belongs to wrong function?

Reply:
The problem was that this function relies on the path of busybox. You should eliminate the expectation of current path. Instead, you should add `popd` at the end of function `build_busybox`. Alternatively, use `make -C ${OUT}/busybox-${BUSYBOX_VER} all install`